time stamp
EyeState left
EyeState right
geometry_msgs/Point32 ball_pos_cam  # in stereo camera coordinates
geometry_msgs/Point32 ball_pos_table  # in table coordinates
time ball_stamp  # time of last ball detection
